jsdoc.json

```diff
@@ -0,0 +1,56 @@
+{
+	"plugins": [ "jsdoc" ],
+	"settings": {
+		"jsdoc": {
+			"tagNamePreference": {
+				"arg": "param",
+				"argument": "param",
+				"class": "class",
+				"tag constructor": "constructor",
+				"augments": "extends",
+				"constant": "const",
+				"defaultvalue": "default",
+				"description": "desc",
+				"emits": "fires",
+				"exception": "throws",
+				"fileoverview": "file",
+				"func": "function",
+				"function": "method",
```

Comment:
both func and function should map to method right?

Reply:
It used to be that "func" mapped to "function", which was very odd, because "function" mapped to "method", but the mapping was not transitive, so both method and function were allowed.

cc @Krinkle @jdforrester

Reply:
Yes.